schemas/com.snowplowanalytics.snowplow.r2f/removal_criteria/jsonschema/1-0-0

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for R2F (Right to be forgotten) criteria files that contain the identifiers for events that should be deleted.",
```

Comment:
should we specify that it's for only one criteria, reading it it made me think it was the schema for the file.

Reply:
Let's change:

> Schema for R2F (Right to be forgotten) criteria files that contain the identifiers for events that should be deleted.

to:

> Schema for an individual entry in an R2F (Right to be forgotten) criteria file, each entry specifying an identifier for events that should be deleted.